python/src/MetropolisHastingsImplementation_doc.i.in

```diff
@@ -321,112 +253,6 @@ OT_MetropolisHastings_getInitialState_doc
 
 // ---------------------------------------------------------------------
 
-%define OT_MetropolisHastings_getAdaptationPeriod_doc
```

Comment:
I understand why `getThinning()` doc is removed: the method is gone. But I do not see why the `getAdaptationPeriod()` doc is removed, since the method looks unchanged to me.

Reply:
Actually, the method was moved to RandomWalkMetropolisHastings, which is why its doc in MetropolisHastingsImplementation is removed.